src/Core/Echo.Ast/Construction/AstParser.cs

```diff
@@ -71,10 +72,26 @@ public ControlFlowGraph<Statement<TInstruction>> Parse()
                 newOrigin.ConnectWith(newTarget, originalEdge.Type);
             }
             
-            // Fix entry point.
+            // Fix entry point(s).
             newGraph.Entrypoint = newGraph.Nodes[_controlFlowGraph.Entrypoint.Offset];
+            FixEntryPoint(_controlFlowGraph);
 
             return newGraph;
+
+            void FixEntryPoint(IControlFlowRegion<TInstruction> region)
+            {
+                foreach (var child in region.GetSubRegions().Where(r => r is {}))
```

Comment:
When does the subregions collection contain `null` objects?

Reply:
Forgot to remove that, my bad :)